apps/src/lib/kits/maker/boards/microBit/MicroBitBoard.js

```diff
@@ -156,6 +158,20 @@ export default class MicroBitBoard extends EventEmitter {
     return !!this.boardClient_.myPort;
   }
 
+  /**
+   * Scroll 'yes' to demonstrate successful connection
+   * to the board.
+   * @returns {Promise} resolved when the song and animation are done.
+   */
+  celebrateSuccessfulConnection() {
+    this.initializeComponents().then(() => {
+      const led = new LedScreen({
+        mb: this.boardClient_
+      });
+      return Promise.resolve().then(() => led.scrollString('yes'));
```

Comment:
The circuit playground celebration flashes its LEDs green and plays a little song. I feel like it's a nice non-verbal way to indicate success. I know most micro:bits won't be connected to a speaker, but I wonder if we could fancy up the visual indicator and not have it rely on English. Maybe a spiral animation that resolves into a checkmark or smiley face? Just throwing the idea out there!

Reply:
Good point about not relying on English. I'll ping Dan S for ideas - as planning to ask him to take a look.  Thanks!